smdebug/core/utils.py

```diff
@@ -319,6 +320,7 @@ def __init__(
         self.tensorboard_dir = tensorboard_dir
         self.training_job_name = training_job_name
         self.json_file_contents = json_file_contents
+        self.disable_tb = disable_tb
```

Comment:
can we change it to enable_tb?
with disable_tb, its like checking not of not

Reply:
Sure.